.github/workflows/overcloud-host-image-build.yml

```diff
@@ -291,9 +292,10 @@ jobs:
           source venvs/kayobe/bin/activate &&
           source src/kayobe-config/kayobe-env --environment ci-builder &&
           kayobe playbook run \
-          src/kayobe-config/etc/kayobe/ansible/pulp-host-image-upload.yml \
+          src/kayobe-config/etc/kayobe/ansible/pulp-image-upload.yml \
           -e image_path=/opt/kayobe/images/overcloud-ubuntu-jammy \
-          -e host_image_tag=${{ steps.host_image_tag.outputs.host_image_tag }} \
+          -e image_tag=${{ steps.host_image_tag.outputs.host_image_tag }} \
+          -e image_name="kayobe-images" \
```

Comment:
image_name is now artifact_path (there are two of these in this file)

Reply:
This needs to be `artifact_name` I think, so we don't conflict with the regex search.